module1/homework/fibonacci/fibonacci.hpp

```diff
@@ -2,10 +2,40 @@
 
 int fibonacci_iterative(int sequence) {
     // TODO: Your implementation goes here
-    return 0;
+    	int a=0;
+	int b=1;
+	int c=0;
+	for (size_t i = 0 ; i < sequence ; i++) {
+	
+
+		if (i < 2){
+		
+			c = 1;
+		} else {
+		
+			a=b;
+			b=c;
+			c=a+b;
```

Comment:
stawiaj spacje przed i po operatorach takich jak = lub +. Kod jest wtedy czytelniejszy.
```suggestion
			a = b;
			b = c;
			c = a + b;
```